tests/test_benchmarking/test_data/test_simulated_data/test_mock_student_provider.py

```diff
@@ -21,6 +21,76 @@ def test_get__returns_student_objects(self):
         for student in students:
             self.assertIsInstance(student, Student)
 
+    def test_get__each_student_has_correct_num_friends_and_enemies(self):
```

Comment:
these tests actually should go in the Helpers test class, since the function that generates students isn't in MockStudentProvider, it is just used there